runtime/ftplugin/man.vim

```diff
@@ -8,14 +8,16 @@ let b:did_ftplugin = 1
 
 setlocal noexpandtab tabstop=8 softtabstop=8 shiftwidth=8
 setlocal wrap breakindent linebreak
+setlocal colorcolumn=0 nolist
 
 " Parentheses and '-' for references like `git-ls-files(1)`; '@' for systemd
 " pages; ':' for Perl and C++ pages.  Here, I intentionally omit the locale
 " specific characters matched by `@`.
 setlocal iskeyword=@-@,:,a-z,A-Z,48-57,_,.,-,(,)
 
-setlocal nonumber norelativenumber
-setlocal foldcolumn=0 colorcolumn=0 nolist nofoldenable
+" man page content is likely preformatted for the terminal width, so
+" narrowing display by any additional columns leads to Embarrassing Line Wrap
+setlocal nonumber norelativenumber foldcolumn=0 signcolumn=auto
```

Comment:
disabling random things is not ideal. instead, find where the width (MANWIDTH, iirc) is calculated in this plugin, and adjust it

Reply:
I don't think `MANWIDTH` is controlled by plugin when calling `man` from CLI. Personally, I have `MANPAGER` set in my environment as described in `:help man`, and just call `man echo` in bash (see steps to reproduce).

Note, that this PR only adds `signcolumn=auto` (may be easier to see when viewing separate commits). Other "random things" (visual columns occupied by editor's UI, to be precise) are already disabled. Suggested changes follow the existing approach and improve it (I believe), not aiming to seek for ideal.

If someone decide to spend time finding an "ideal" solution, they could still benefit from these changes by using them as a reference. E. g. just like they will see "nonumber", "norelativenumber", "foldcolumn" and include the width of these columns into their calculations of MANWIDTH (or whatever is relevant), they will also see the "signcolumn" and remember to take it into account as well.

Also, IMHO, benefits of what you call an "ideal" approach are questionable. ftplugins are specifically intended to improve user experience with specific `filetype`s by taking into account the meaning of the file type. Disabling/enabling specific things for specific filetypes feels totally reasonable for me. Do you think that other `setlocal` options <s>are not ideal</s> should be avoided in this ftplugin (e. g. `nolist`)?

Reply:
This is fine, since we already were doing `nonumber norelativenumber`, and :Man always opens a new window anyway (otherwise restoring window-local options is tricky).

why signcolumn=auto instead of signcolumn=no ?

Reply:
@justinmk 
> why auto instead of no

my reasoning was: as far as I know, sings are not used in Vim *by default*, and therefore they only appear when user **explicitly configures them** (e. g. installs a plugin for that purpose). I only know about plugins that use signs for editable files (integration of git repos) or specific filetypes (integration of linters, compilers, build systems). So:
- in all situations that I can imagine, signcolumn is empty anyway in ft=man, so no difference between `auto` and `no`
- if by any chance there ARE some signs in singcolumn for ft=man, then user must have made an explicit **effort** to add them, and they could be annoyed by the fact that signs are still not visible.

On the other side, the bad scenario for this decision (using auto instead of no) is when user (1) has signcolumn=no in their global config AND (2) still creates some signs when browsing man pages (maybe unintentionally, with a plugin). It may be annoying for them to see signcolumn appear given that they have explicitly disabled it in `init.vim`. Regarding this case:
1. I'm fine with signcolumn=no if anyone believes it is better.
2. A more clever solution could be to check the current value of signcolumn and only downgrade from "yes" to "auto", but keep the original value otherwise. This is more in line with the motivation of the proposed change — to prevent signcolumn from being "yes" (which may be good as a global value, but bad for ft=man).

I'm equally ready to implement any of two approaches above or keep the current one.